sdk/testing/src/main/java/io/opentelemetry/sdk/testing/junit4/OpenTelemetryRule.java

```diff
@@ -68,8 +68,6 @@ public static OpenTelemetryRule create() {
   private final OpenTelemetrySdk openTelemetry;
   private final InMemorySpanExporter spanExporter;
 
-  private volatile OpenTelemetry previousGlobalOpenTelemetry;
```

Comment:
With SPI gone and double-registration blocked, the global OTel is explicitly configured, and needs to be explicitly reset, so we don't need this backing up anymore.